test/common/matchers.dart

```diff
@@ -5,6 +5,7 @@ import 'package:test/test.dart';
 
 import 'package:build/build.dart';
 
+final argumentError = new isInstanceOf<ArgumentError>();
```

Comment:
Actually, you can replace `throwsA(argumentError)` with `throwsArgumentError` from package:test

https://github.com/dart-lang/test/blob/master/lib/src/frontend/throws_matchers.dart#L10